.github/workflows/ci.yml

```diff
@@ -13,6 +18,7 @@ concurrency:
 jobs:
   build:
     name: Build
+    if: ${{ ( github.event_name != 'schedule' ) || ( github.event_name == 'schedule' && github.repository == 'keycloak/keycloak' ) }}
```

Comment:
I hate that this is the only way to do this. GitHub really should have a way to put conditions/filters on the events, not only on individual jobs, but it is what it is

Reply:
I agree, but that's what we have as of today ...